### PR TITLE
Microsoft.Data.Sqlite: Test on .NET Framework

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: test/Microsoft.Data.Sqlite.Tests/Microsoft.Data.Sqlite.Tests.csproj [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net461</TargetFrameworks>
     <DefineConstants>$(DefineConstants);E_SQLITE3</DefineConstants>
     <Nullable>enable</Nullable>
+    <UseCurrentRuntimeIdentifier Condition="'$(TargetFramework)' == 'net461'">True</UseCurrentRuntimeIdentifier>
```
Comment (bricelam) [inline test/Microsoft.Data.Sqlite.Tests/Microsoft.Data.Sqlite.Tests.csproj:7]: Needed for NuGet to resolve native SQLitePCLRaw assets

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs [hunk 2]
```diff
@@ -44,7 +47,42 @@ public partial class SqliteConnection : DbConnection
         private static readonly StateChangeEventArgs _fromOpenToClosedEventArgs = new StateChangeEventArgs(ConnectionState.Open, ConnectionState.Closed);
 
         static SqliteConnection()
-            => BundleInitializer.Initialize();
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs:47]: Simplified code and inlined it. No need to cache all those values--it only runs once

**Review by smitpatel — APPROVED** [review]

File: test/Microsoft.Data.Sqlite.Tests/Microsoft.Data.Sqlite.Tests.csproj (hunk 1 above)
Comment (bricelam) [reply test/Microsoft.Data.Sqlite.Tests/Microsoft.Data.Sqlite.Tests.csproj:7]: Gonna update SQLitePCLRaw and remove these

File: benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj [hunk 3]
```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.6" PrivateAssets="None" />
+    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.7" PrivateAssets="None" />
```
Comment (bricelam) [inline benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj:19]: @wtgodbe Can we mirror the 2.0.7 versions of the following packages?

- SQLitePCLRaw.bundle_e_sqlcipher
- SQLitePCLRaw.bundle_e_sqlite3
- SQLitePCLRaw.bundle_sqlite3
- SQLitePCLRaw.bundle_winsqlite3
- SQLitePCLRaw.core
- SQLitePCLRaw.lib.e_sqlcipher
- SQLitePCLRaw.lib.e_sqlite3
- SQLitePCLRaw.provider.dynamic_cdecl
- SQLitePCLRaw.provider.e_sqlcipher
- SQLitePCLRaw.provider.e_sqlite3
- SQLitePCLRaw.provider.sqlite3
- SQLitePCLRaw.provider.winsqlite3

Comment (wtgodbe) [reply benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj:19]: If I'm not around, you can ask First Responders/dnceng to mirror packages in the future

**Pull request merged**